src/lib/configs.ts

```diff
@@ -102,4 +102,31 @@ export function getConfigs(): { [key: string]: string } {
     GLEE_CONFIG_FILE_PATH,
     ASYNCAPI_FILE_PATH
   }
-}
\ No newline at end of file
+}
+
+
+export class GlobalGleeConfig {
+  private _config: GleeConfig
+
+  constructor(config: GleeConfig) {
+    this._config = config
+  }
+
+  async init(protocol: string){
+
+    const defaultValues = {
+      docs: {
+        enabled: true,
+        folder: 'docs',
+        template: '@asyncapi/markdown-template'
+      }
+    }
```

Comment:
We can define the default values like this in the init function where we can basically return default values for the config or the passed values.